aws-logging-cloudwatch/src/main/java/com/amplifyframework/logging/cloudwatch/models/CloudWatchLogEvent.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amplifyframework.logging.cloudwatch.models
+
+internal data class CloudWatchLogEvent(val timestamp: Long, val message: String, val id: Int = 0)
```

Comment:
I think id should be a long here according to Sqlite integer primary keys

Reply:
I also don't like the double usage here making id a default value of 0, but also pulling a CloudWatchLogEvent from db that has a real id. These should be split up.